idaes/generic_models/flowsheets/tests/test_methanol_flowsheet.py

```diff
@@ -0,0 +1,193 @@
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Tests for methanol flowsheet.
+
+"""
+
+import pytest
+
+from idaes.generic_models.flowsheets.methanol_flowsheet import (
+    build_model, set_inputs, initialize_flowsheet, add_costing, report)
+
+from pyomo.environ import (Constraint,
+                           ConcreteModel,
+                           value)
+from pyomo.network import Arc
+from pyomo.environ import TerminationCondition
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.util import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+
+from idaes.generic_models.properties.core.generic.generic_property import \
+    GenericParameterBlock
+from idaes.generic_models.properties.core.generic.generic_reaction import \
+    GenericReactionParameterBlock
+
+from idaes.generic_models.unit_models import (
+    Mixer,
+    Heater,
+    Compressor,
+    Turbine,
+    StoichiometricReactor,
+    Flash)
+
+
+@pytest.fixture(scope='module')
+def model():
+    m = ConcreteModel()
+    m = build_model(m)
+
+    return m
+
+
+@pytest.mark.unit
+def test_build_flowsheet(model):
+    assert isinstance(model.fs, FlowsheetBlock)
+
+    assert isinstance(model.fs.thermo_params, GenericParameterBlock)
+    assert isinstance(model.fs.reaction_params, GenericReactionParameterBlock)
+
+    assert isinstance(model.fs.M101, Mixer)
+    assert isinstance(model.fs.C101, Compressor)
+    assert isinstance(model.fs.H101, Heater)
+    assert isinstance(model.fs.R101, StoichiometricReactor)
+    assert isinstance(model.fs.T101, Turbine)
+    assert isinstance(model.fs.H102, Heater)
+    assert isinstance(model.fs.F101, Flash)
+
+    assert isinstance(model.fs.s02, Arc)
+    assert isinstance(model.fs.s03, Arc)
+    assert isinstance(model.fs.s04, Arc)
+    assert isinstance(model.fs.s05, Arc)
+    assert isinstance(model.fs.s06, Arc)
+    assert isinstance(model.fs.s07, Arc)
+
+    assert degrees_of_freedom(model) == 25
+
+
+@pytest.mark.unit
+def test_set_inputs(model):
+    set_inputs(model)
+
+    assert degrees_of_freedom(model) == 0
+
+
+@pytest.mark.unit
+def test_initialize_flowsheet(model):
+    initialize_flowsheet(model)
+
+    assert degrees_of_freedom(model) == 0
+
+    assert model.fs.M101.outlet.flow_mol[0].value == pytest.approx(954, 1e-3)
```

Comment:
Do you have a source for these values, or are they just the values you got when you ran the model? If you have a source, it would be good to report it here.